packages/ndla-ui/src/ResourceBox/ResourceBox.tsx

```diff
@@ -0,0 +1,170 @@
+/**
+ * Copyright (c) 2022-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+import React from 'react';
+import { breakpoints, fonts, mq, colors } from '@ndla/core';
+import { useTranslation } from 'react-i18next';
+import { Launch } from '@ndla/icons/common';
+import { LicenseByline } from '@ndla/licenses';
+import { SafeLinkButton } from '@ndla/safelink';
+import styled from '@emotion/styled';
+import Image from '../Image';
+
+const BoxWrapper = styled.div`
+  height: 192px;
+  border-radius: 5px;
+  border: 1px solid #deebf6;
+  position: relative;
+  right: auto;
+  left: -16.6666666667%;
+  width: 133.3333333333%;
+
+  margin-bottom: 24px;
+  font-family: ${fonts.sans};
+  box-shadow: 0px 20px 35px -15px rgba(32, 88, 143, 0.15);
+  ${mq.range({ until: breakpoints.tabletWide })} {
+    margin: 0 auto;
+    width: 80%;
+    left: 0;
+    padding-left: 24px;
+    padding-right: 24px;
+    margin-bottom: 24px;
+    height: 500px;
+  }
+`;
+
+const Boxtitle = styled.h3`
+  font-weight: ${fonts.weight.bold};
+  font-size: ${fonts.sizes(18)};
+  ${mq.range({ until: breakpoints.tabletWide })} {
+    margin-top: 35px;
+  }
+`;
+
+const Boxcaption = styled.p`
+  font-size: ${fonts.sizes(14)};
+  ${mq.range({ until: breakpoints.tabletWide })} {
+    line-height: 22px;
+  }
+`;
+
+const StyledButtonDiv = styled.div`
+  position: absolute;
+  bottom: 18%;
+  left: 25%;
+  text-align: center;
+  ${mq.range({ until: breakpoints.tabletWide })} {
+    bottom: 5%;
+    left: 0;
+    display: flex;
+    justify-content: center;
+    width: 100%;
+  }
+`;
+const NewStyledButton = styled(SafeLinkButton)`
```

Comment:
Må sei at eg ikkje er veldig glad i navn med New i seg, for dei vil snart være gamle. Bruk heller navn som beskriver funksjonen bedre.

Reply:
Så klart, det er god konvensjon å ikke bruke new.